util/fees_test.go

```diff
@@ -0,0 +1,83 @@
+package util
+
+import (
+	"bytes"
+	"github.com/OpenBazaar/wallet-interface"
+	"testing"
+)
+
+type ClosingBuffer struct {
+	*bytes.Buffer
+}
+
+func (cb *ClosingBuffer) Close() (err error) {
+	return
+}
+
+type mockExchangeRate struct{}
+
+var returnRate float64 = 438
+
+func (m *mockExchangeRate) GetExchangeRate(currencyCode string) (float64, error) {
+	return 0, nil
+}
+
+func (m *mockExchangeRate) GetLatestRate(currencyCode string) (float64, error) {
+	return returnRate, nil
+}
+
+func (m *mockExchangeRate) GetAllRates(usecache bool) (map[string]float64, error) {
+	return make(map[string]float64), nil
+}
+
+func (m *mockExchangeRate) UnitsPerCoin() int {
+	return 0
+}
+
+func TestFeeProvider_GetFeePerByte(t *testing.T) {
+	fp := NewFeeProvider(2000, 360, 320, 280, &mockExchangeRate{})
+
+	// Test using exchange rates
+	if fp.GetFeePerByte(wallet.PRIOIRTY) != 50 {
```

Comment:
I don't see how the 360 amount turns into 50 given the mockExchangeRateProvider doesn't have any rates for USD.

Reply:
It was in the `returnedRate` var. I've move the var inside the struct.

Reply:
It looks as though the mockExchangeRate doesn't have any rates... which leads me to think it's using the default fee?